pyproject.toml

```diff
@@ -17,42 +17,39 @@ dynamic = ["version"]
 
 requires-python = ">=3.8"
 dependencies = [
+    "astropy",
+    "fsspec", # Used for abstract filesystems
     "healpy",
+    "numba",
     "pandas",
-    "setuptools_scm",
     "pyarrow>=10.0.0",
-    "astropy",
-    "typing-extensions>=4.3.0",
-    "numba",
-    "fsspec", # Used for abstract filesystems
-    "ipykernel", # Support for Jupyter notebooks
+    "setuptools_scm",
```

Comment:
Why should it be a runtime-dependency, not a build-time one?

Reply:
We do something silly in this package (and hipscat-import) to fetch the release number of the running library, so we can write it in the provenance info file. It was depending on setuptools at some point (but might not anymore). I'm scared to try to remove it now. Can you let me try that in a separate PR?

Reply:
Sure, thank you. Theoretically, `importlib.metadata.version` should work...